### ISSUE TITLE
Favorites and Shares icon in sidebar misaligned

### ISSUE THREAD

**Issue by MorrisJobke** [nextcloud/server#10368]
The icons for the categories "Favorites" and "Shares" are not there or at the wrong position.

![bildschirmfoto 2018-07-24 um 11 17 34](https://user-images.githubusercontent.com/245432/43129396-96288022-8f34-11e8-9f6b-39e03baa9f29.png)

cc @jancborchardt @skjnldsv @newhinton 

Maybe because of #9982

**Comment by skjnldsv** [comment]
I cannot reproduce :/
@nextcloud/designers

**Comment by MorrisJobke** [comment]
> The icons for the categories "Favorites" and "Shares" are not there or at the wrong position.

Not there was fixed in #10378

Now it looks like this in Safari:

![bildschirmfoto 2018-07-25 um 09 36 37](https://user-images.githubusercontent.com/245432/43186179-41ed0662-8fee-11e8-91f0-8dba932d45a1.png)

Works fine in Firefox and Chrome. It is this transition rule:

```css
#app-navigation .collapsible:before, #app-navigation .collapsible > a {
    transition: background 100ms ease-in-out, transform 250ms ease-in-out, opacity 100ms ease-in-out;
}
```

Once the part `background 100ms ease-in-out` is removed it works again on Safari as well.

**Comment by skjnldsv** [comment]
Fix in #10386
